### DISCUSSION TITLE
Does this package also run on Next js (ts)?

### DISCUSSION THREAD

**Discussion by 9mean2** [k-yle/rtsp-relay#255] [Q&A]
Does this package run in the same way as React in Next js?

And if you can connect 2 rtsp addresses? I want to show two cameras on the screen.

**Comment by k-yle** [comment]
You can't use the [`<canvas>`](https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API) element in SSR, because it's a client-only tag. 

Assuming you render everything stream-related on the client side, then it should work fine.

**Reply by 9mean2** [reply]
Also, if do it in React, is it also possible to connect two cameras?

For example, rtsp address 1 on the left canvas and rtsp address 2 on the right canvas.

**Reply by k-yle** [reply]
Yes, you can render multiple canvas elements. There is a short section in the README that shows how to configure the backend for a flexible number of cameras
